torchvision/prototype/models/detection/keypoint_rcnn.py

```diff
@@ -56,15 +56,15 @@ class KeypointRCNN_ResNet50_FPN_Weights(WeightsEnum):
             "map_kp": 65.0,
         },
     )
-    default = Coco_V1
+    default = COCO_V1
 
 
 @handle_legacy_interface(
     weights=(
         "pretrained",
         lambda kwargs: KeypointRCNN_ResNet50_FPN_Weights.Coco_Legacy
```

Comment:
`Coco_Legacy` was missed

Reply:
I'm not sure what it should be modified to, maybe `COCO_Legacy` ?

Reply:
Probably the latter. All caps. This is part of the reason why ideally we should have these changes on a single PR, so that we can see all these corner cases as a whole and decide a common approach.